datastore/tests/test_additional_data.py

```diff
@@ -0,0 +1,43 @@
+from django.test import TestCase
```

Comment:
I was thinking about this test file. Do you think we could instead of a file have a folder of `additional_data` and inside it a file per source?

Reply:
I'd prefer we kept all the tests in the tests module, the reason for this is that the tests are cross application/cross module tests, for example the test requires db.models . If it was entirely independent then yeah possibly a case for having them per module. I prefer integration style tests as they generate better coverage.

Reply:
It is not about what type of test we write. I was making the suggestion not to have everything in one file that could be very long. We can leave it as it is but I think I will create a separate file per source.